line-colors.csv

```diff
@@ -94,6 +94,25 @@ db-regionetz-westfrankenbahn,RE 80,db-regionetz-verkehrs-gmbh-westfrankenbahn,re
 db-regionetz-westfrankenbahn,RE 87,db-regionetz-verkehrs-gmbh-westfrankenbahn,re-87,#ff2e17,#ffffff,,rectangle
 ding-swu,STR 1,,8-ald087-1,#e52329,#ffffff,,rectangle
 ding-swu,STR 2,,8-ald087-2,#3fad56,#ffffff,,rectangle
+dvg-dessau,1,,8-nasdvg-1,#f59c00,#ffffff,,rectangle
+dvg-dessau,3,,8-nasdvg-3,#f59c00,#ffffff,,rectangle
+dvg-dessau,10,,5-nas001-10,#2e76bb,#ffffff,,pill
+dvg-dessau,11,,5-nas001-11,#95d2f1,#ffffff,,pill
+dvg-dessau,12,,5-nas001-12,#529bb9,#ffffff,,pill
+dvg-dessau,13,,5-nas001-13,#9ec753,#ffffff,,pill
+dvg-dessau,14,,5-nas001-14,#e3000b,#ffffff,,pill
+dvg-dessau,15,,5-nas001-15,#a80126,#ffffff,,pill
+dvg-dessau,16,,5-nas001-16,#941680,#ffffff,,pill
+dvg-dessau,17,,5-nas001-16,#ffcc00,#ffffff,,pill
+dvg-dessau,21,,5-nas001-16,#ee722e,#ffffff,,pill
+dvg-dessau,22,,5-nas001-16,#0f3f93,#ffffff,,pill
+dvg-dessau,N1,,5-nas001-n1,#000000,#ffffff,,pill
+dvg-dessau,N2,,5-nas001-n2,#000000,#ffffff,,pill
+dvg-dessau,N3,,5-nas001-n3,#000000,#ffffff,,pill
+dvg-dessau,N4,,5-nas001-n4,#000000,#ffffff,,pill
+dvg-dessau,N5,,5-nas001-n5,#000000,#ffffff,,pill
+dvg-dessau,N6,,5-nas001-n6,#000000,#ffffff,,pill
+dwe-dessau,DWE,,dwe,#ffffff,#000000,#000000,rectangle
```

Comment:
This operator is missing in the sources.json file!

Reply:
my bad, sorry!